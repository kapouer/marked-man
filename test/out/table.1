TABLE(7)                                                              TABLE(7)

NNAAMMEE
       ttaabbllee - test data for table format

       test table copy and paste form www.

LLAARRGGEE TTAABBLLEE
       _C_o_m_p_a_r_i_s_o_n                _o_f                _f_i_l_e                _s_y_s_t_e_m_s
       ⟨https://en.wikipedia.org/wiki/Comparison_of_file_systems⟩

       ┌──────────────────────────────────────────────────────────────────────┐
       │File system      Creator           Year  of  intro‐  Original operat‐ │
       │                                   duction           ing system       │
       ├──────────────────────────────────────────────────────────────────────┤
       │DECtape          DEC               1964              PDP-6 Monitor    │
       ├──────────────────────────────────────────────────────────────────────┤
       │DASD             IBM               1964              OS/360           │
       ├──────────────────────────────────────────────────────────────────────┤
       │Level-D          DEC               1968              TOPS-10          │
       ├──────────────────────────────────────────────────────────────────────┤
       │George 2         ICT (later ICL)   1968              George 2         │
       ├──────────────────────────────────────────────────────────────────────┤
       │Version  6  Unix Bell Labs         1972              Version 6 Unix   │
       │file      system                                                      │
       │(V6FS)                                                                │
       ├──────────────────────────────────────────────────────────────────────┤
       │RT-11 file  sys‐ DEC               1973              RT-11            │
       │tem                                                                   │
       ├──────────────────────────────────────────────────────────────────────┤
       │Disk   Operating GEC               1973              Core   Operating │
       │System (GEC DOS)                                     System           │
       ├──────────────────────────────────────────────────────────────────────┤
       │CP/M file system Digital Research  1974              CP/M[1]          │
       │                 (Gary Kildall)                                       │
       ├──────────────────────────────────────────────────────────────────────┤
       │ODS-1            DEC               1975              RSX-11           │
       ├──────────────────────────────────────────────────────────────────────┤
       │GEC  DOS  filing GEC               1977              OS4000           │
       │system extended                                                       │
       ├──────────────────────────────────────────────────────────────────────┤
       │FAT (8-bit)      Microsoft  (Marc  1977              Microsoft Stand‐ │
       │                 McDonald)    for                    alone  Disk  BA‐ │
       │                 NCR                                 SIC-80    (later │
       │                                                     Microsoft Stand‐ │
       │                                                     alone  Disk  BA‐ │
       │                                                     SIC-86)          │
       ├──────────────────────────────────────────────────────────────────────┤
       │DOS 3.x          Apple Computer    1978              Apple DOS        │
       ├──────────────────────────────────────────────────────────────────────┤
       │Pascal           Apple Computer    1978              Apple Pascal     │
       ├──────────────────────────────────────────────────────────────────────┤
       │CBM DOS          Commodore         1978              Commodore BASIC  │
       ├──────────────────────────────────────────────────────────────────────┤
       │Version  7  Unix Bell Labs         1979              Version 7 Unix   │
       │file      system                                                      │
       │(V7FS)                                                                │
       ├──────────────────────────────────────────────────────────────────────┤
       │ODS-2            DEC               1979              OpenVMS          │
       └──────────────────────────────────────────────────────────────────────┘
LLOONNGG TTEEXXTT
       ┌──────────────────────────────────────────────────────────────────────┐
       │ Person                Work                      Quote                │
       ├──────────────────────────────────────────────────────────────────────┤
       │ Sherlock Holmes       The Norwood Builder       You mentioned  your  │
       │                                                 name as if I should  │
       │                                                 recognize  it,  but  │
       │                                                 beyond  the obvious  │
       │                                                 facts that you  are  │
       │                                                 a  bachelor,  a so‐  │
       │                                                 licitor, a  freema‐  │
       │                                                 son,  and  an asth‐  │
       │                                                 matic, I know noth‐  │
       │                                                 ing  whatever about  │
       │                                                 you.                 │
       └──────────────────────────────────────────────────────────────────────┘
TTHHIINNGG TTAABBLLEE
       ┌──────────────────────────────────────────────────────────────────────┐
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            T{1                                        2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       ├──────────────────────────────────────────────────────────────────────┤
       │            1                                          2              │
       └──────────────────────────────────────────────────────────────────────┘

                                 January 1979                         TABLE(7)
