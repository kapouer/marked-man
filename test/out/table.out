[4mTABLE[24m(7)               Miscellaneous Information Manual               [4mTABLE[24m(7)

[1mNAME[0m
       [1mtable [22m- test data for table format

       test table copy and paste form www.

[1mLARGE TABLE[0m
       ]8;;https://en.wikipedia.org/wiki/Comparison_of_file_systems\[4mComparison[24m [4mof[24m [4mfile[24m [4msystems[24m]8;;\
       ┌─────────────────────────────────────────────────────────────────────────┐
       │ File system       Creator           Year  of  intro‐   Original operat‐ │
       │                                     duction            ing system       │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ DECtape           DEC               1964               PDP-6 Monitor    │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ DASD              IBM               1964               OS/360           │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ Level-D           DEC               1968               TOPS-10          │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ George 2          ICT (later ICL)   1968               George 2         │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ Version  6  Unix  Bell Labs         1972               Version 6 Unix   │
       │ file      system                                                        │
       │ (V6FS)                                                                  │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ RT-11 file  sys‐  DEC               1973               RT-11            │
       │ tem                                                                     │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ Disk   Operating  GEC               1973               Core   Operating │
       │ System (GEC DOS)                                       System           │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ CP/M file system  Digital Research  1974               CP/M[1]          │
       │                   (Gary Kildall)                                        │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ ODS-1             DEC               1975               RSX-11           │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ GEC  DOS  filing  GEC               1977               OS4000           │
       │ system extended                                                         │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ FAT (8-bit)       Microsoft  (Marc  1977               Microsoft Stand‐ │
       │                   McDonald)    for                     alone  Disk  BA‐ │
       │                   NCR                                  SIC-80    (later │
       │                                                        Microsoft Stand‐ │
       │                                                        alone  Disk  BA‐ │
       │                                                        SIC-86)          │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ DOS 3.x           Apple Computer    1978               Apple DOS        │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ Pascal            Apple Computer    1978               Apple Pascal     │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ CBM DOS           Commodore         1978               Commodore BASIC  │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ Version  7  Unix  Bell Labs         1979               Version 7 Unix   │
       │ file      system                                                        │
       │ (V7FS)                                                                  │
       ├─────────────────────────────────────────────────────────────────────────┤
       │ ODS-2             DEC               1979               OpenVMS          │
       └─────────────────────────────────────────────────────────────────────────┘

[1mLONG TEXT[0m
       ┌────────────────────────────────────────────────────────────────────┐
       │ Person                Work                     Quote               │
       ├────────────────────────────────────────────────────────────────────┤
       │ Sherlock Holmes       The Norwood Builder      You  mentioned your │
       │                                                name as if I should │
       │                                                recognize  it,  but │
       │                                                beyond  the obvious │
       │                                                facts that you  are │
       │                                                a  bachelor,  a so‐ │
       │                                                licitor, a  freema‐ │
       │                                                son,  and  an asth‐ │
       │                                                matic, I know noth‐ │
       │                                                ing whatever  about │
       │                                                you.                │
       └────────────────────────────────────────────────────────────────────┘

[1mTHING TABLE[0m
       ┌──────────────────────────────────────────────┐
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ T{1                                        2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       ├──────────────────────────────────────────────┤
       │ 1                                          2 │
       └──────────────────────────────────────────────┘

                                 January 1979                         [4mTABLE[24m(7)
